src/extractors/custom/www.publickey1.jp/index.js

```diff
@@ -0,0 +1,29 @@
+export const WwwPublickey1JpExtractor = {
+  domain: 'www.publickey1.jp',
+
+  title: {
+    selectors: ['h1'],
+  },
+
+  author: {
+    selectors: ['#subcol p:has(img)'],
+  },
+
+  date_published: null,
```

Comment:
You could query the date using `.pubdate` as a selector, in addition to a `format` key in order to define and parse this date format that contains special characters.
So maybe something like:
```javascript
  date_published: {
    selectors: ['.pubdate'],
    format: 'YYYY年MM月DD日',
  },
```